lib/device_map.py

```diff
@@ -612,6 +612,17 @@
                'fastboot': False,
                'mach': 'renesas'}
 
+r8a7796_m3ulcb = {'device_type': 'r8a7796-m3ulcb',
+               'templates': [ 'generic-arm64-uboot-image-dtb-ramdisk-template.jinja2'],
+               'kernel_defconfig_blacklist': [],
+               'defconfig_blacklist': ['arm64-allnoconfig',
```

Comment:
We've found an issue with the blacklisting, so please update this to just be "allnoconfig" (drop the arch)